var/spack/repos/builtin/packages/py-onnx/package.py

```diff
@@ -42,8 +44,12 @@ class PyOnnx(PythonPackage):
     depends_on("py-protobuf@3.12.2:", type=("build", "run"), when="@1.11.0")
     depends_on("protobuf@3.12.2:3.20.1", when="@1.12.0:")
     depends_on("py-protobuf@3.12.2:3.20.1", type=("build", "run"), when="@1.12.0:")
+    depends_on("protobuf@3.20.2:", when="@1.13.0:")
+    depends_on("py-protobuf@3.20.2:", type=("build", "run"), when="@1.13.0:")
     depends_on("py-numpy", type=("build", "run"))
     depends_on("py-numpy@1.16.6:", type=("build", "run"), when="@1.8.1:")
+    depends_on("py-numpy@1.21.5:", type=("build", "run"), when="@1.13:")
+    depends_on("py-numpy@1.23.2:", type=("build", "run"), when="@1.13: ^python@3.11:")
```

Comment:
According to `requirements.txt`, it's still numpy >= 1.16.6

Reply:
Right, was looking at main, and https://github.com/onnx/onnx/pull/5095